coopr-docs/docs/source/installation/ui-config.rst

```diff
@@ -13,33 +13,70 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-:orphan:
-
 .. index::
-single: UI Configuration
+   single: UI Configuration
 
 ================
 UI Configuration
 ================
 
+SSL Configuration Steps
+=======================
+
+SSL configuration requires these steps:
+
+- Create a SSL certificate and key 
+- Put the certificate and key in position
+- Set the environment variables shown below, to describe the locations
+- Set and export the variables
+- Correctly configure the UI
+
+
 Running UI with SSL
-^^^^^^^^^^^^^^^^^^^^^^^
+===================
 
-To enable running UI with SSL, setup these environment variables:
+To enable running the UI with SSL, set these environment variables:
 
 ====================================     ==========================    =======================================
    Environment variable                     Default Value                     Description
 ====================================     ==========================    =======================================
-COOPR_UI_KEY_FILE                        COOPR_HOME/cert/server.key     Key file location.
-COOPR_UI_CERT_FILE                       COOPR_HOME/cert/server.crt     Certificate password.
+COOPR_UI_KEY_FILE                        COOPR_HOME/cert/server.key    Key file location
+COOPR_UI_CERT_FILE                       COOPR_HOME/cert/server.crt    Certificate file location
+COOPR_UI_SSL_PORT                        8443                          Port the UI listens on for connections  
 ====================================     ==========================    =======================================
 
-To configure UI to support server that uses mutual authentication with SSL, setup these environment variables:
+To configure the UI to support a server that uses mutual authentication with SSL, set these 
```

Comment:
We need to remove all references to mutual auth with SSL in coopr 0.9.9 documentation.
For this file, we should remove lines 48-57.